Chase Board/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html5>
+<html>
+	<head>
+    <title>Chase Board</title>
```

Comment:
Change title - chess board
And while adding htmls, please add screenshot of UI

Reply:
Done and added screenshots for all HTML pages